server/api.js

```diff
@@ -135,15 +135,17 @@ router.post("/publish-tenders", (req, res) => {
 	res.status(200).json({ message: "Form submitted successfully!" });
 });
 
-router.get("/buyer-Tender", async (req, res) => {
-	// const buyerId = req.params.id;
+router.get("/buyer-Tender/page", async (req, res) => {
+	const buyerId = 1;
+	const page = req.params.page;
+	const itemsPerPage = 25;
 
-	const result = await db.query("SELECT * FROM tender WHERE buyer_id limit = 25");
+	const result = await db.query(`SELECT * FROM buyer WHERE buyerId= ${buyerId} LIMIT ${itemsPerPage} OFFSET ${(page - 1) * itemsPerPage}`);
```

Comment:
Did you test this? There is no field named `buyerId`.

Also, this is unsafe. You should be using parameterized queries.

Reply:
I have updated safy issue. But haven't tested it.